AMP.md

```diff
@@ -0,0 +1,31 @@
+#Accelerated Mobile Pages.
```

Comment:
The readMe is currently outputting the `#` with all of the headlines. They all need a space between the last `#` and the headline text. (Click the "Display the rich diff" button above to view this.)

Reply:
Fixed